lib/resources/http.rb

```diff
@@ -25,13 +25,14 @@ class Http < Inspec.resource(1)
     "
 
     # rubocop:disable ParameterLists
-    def initialize(url, method: 'GET', params: nil, auth: {}, headers: {}, data: nil)
+    def initialize(url, method: 'GET', params: nil, auth: {}, headers: {}, data: nil, ssl_verify: true)
```

Comment:
I would prefer to use `ssl: { verify: true }` as input, since we do not need to do a lot of translation between our own config and faraday config. Also this would make the path to implement the improvements proposed in #1649


I propose we hand over a ssl hash entry and pass this. Right now, we support verify only.